yarn.lock

```diff
@@ -9795,13 +9813,13 @@ react-native-linear-gradient@2.4.0:
   dependencies:
     prop-types "^15.5.10"
 
-react-native-maps@expo/react-native-maps#v0.22.1-exp.0:
+"react-native-maps@github:expo/react-native-maps#v0.22.1-exp.0":
   version "0.22.1"
   resolved "https://codeload.github.com/expo/react-native-maps/tar.gz/e6f98ff7272e5d0a7fe974a41f28593af2d77bb2"
 
-"react-native-multi-slider@https://github.com/kiwicom/react-native-multi-slider.git#beb71bf5eb210fed393cb29b18032105b725611e":
+"react-native-multi-slider@git+https://github.com/kiwicom/react-native-multi-slider.git#beb71bf5eb210fed393cb29b18032105b725611e":
```

Comment:
<img width="864" alt="screenshot 2019-01-31 at 10 15 44" src="https://user-images.githubusercontent.com/2660330/52044338-3fcd5000-2542-11e9-84f6-0e997d116498.png">

After running `yarn install` I see same `yarn.lock` difference that we've tried to solve yesterday.
We can probably merge it and try to solve it in different PR.
Or maybe can you try to edit it manually and run `yarn install` if those `git+` and `github:` will return?

Reply:
> Or maybe can you try to edit it manually

That sounds like a really bad idea.

Also, techincally, `git +` is correct [when installing npm dependency from git](https://stackoverflow.com/questions/17509669/how-to-install-an-npm-package-from-github-directly#answer-17509764)

Reply:
Ok, I was mostly just curious if `yarn install` on your machine will put it back again.
Sadly if I delete my `yarn.lock` and try to create fresh one `git+` is correctly there in mentioned dependencies, but I'm getting even more version differences.

Reply:
this whole `yarn.lock` is confusing 😞 

I did `rm yarn.lock && yarn install` and I had changes to yarn lock. I repeated and then did just `yarn install` and now it does not change. So I pushed the latest version

Reply:
Now if I run `yarn install` I see same three differences.
But if I delete `yarn.lock` the fresh one is ok without any difference. So the "solution" can be to force everyone with this problem to create fresh `yarn.lock` so we can move forward. But it seems to be little bit fragile and I'm slightly worried that it will somehow return again later.

Reply:
Well, the current `yarn.lock` is not working well on the CI, there is a lot of failures without reason. On this branch, the CI has not yet failed once.

Reply:
So I vote for merge this branch.

Reply:
I'm also for merging of this branch. But main problem right now is that if I create fresh project from this branch and run `yarn install` I'll get those three differences. And we cannot ask everyone who will try to run this project just to delete `yarn.lock` first.
Damn 😄 , this whole problem doesn't really make sense, when correct file is only created when there is no current one.

Reply:
let's add "yarn.lock" to git ignore :D

Reply:
We are not only one with this problem https://github.com/yarnpkg/yarn/issues/4379#issuecomment-334948680

Reply:
So I think that solution is we can add  `--frozen-lockfile` `true` to `.yarnrc`
But as I think it is not necessary for now and If yarn.lock is updated with just yarn install but the app works it is OK.

Reply:
But we should change CI settings to `yarn install --frozen-lockfile`

Reply:
What do you think? https://github.com/kiwicom/margarita/pull/176

Reply:
@tbergq can you rebase it pls?

Reply:
🚀